render/render.js

```diff
@@ -747,7 +747,7 @@ module.exports = function($window) {
 		}
 	}
 	function isFormAttribute(vnode, attr) {
-		return attr === "value" || attr === "checked" || attr === "selectedIndex" || attr === "selected" && vnode.dom === $doc.activeElement || vnode.tag === "option" && vnode.dom.parentNode === $doc.activeElement
```

Comment:
Are we sure removing this `activeElement` check makes sense? It seems to be associated to changing the `selected` attr. Do we know what this check was trying to achieve? I don't think it's related to #2256 
Here is the old condition with parentheses to clarify it. 
```js
return (
  attr === 'value' ||
  attr === 'checked' ||
  attr === 'selectedIndex' ||
  (attr === 'selected' && vnode.dom === $doc.activeElement) ||
  (vnode.tag === 'option' && vnode.dom.parentNode === $doc.activeElement)
);
```